package-loader-rule.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+
+const packages = [
+    'core',
+    'apple-pay',
+    'payment-integration',
+];
```

Comment:
Would there be a more explicit way for a new package to be added? Seems a bit hidden for a new contributor to add new packages